synapse/lib/stormtypes.py

```diff
@@ -841,6 +842,46 @@ def _reqStr(self, name):
                 ret.append((key, valu))
         return ret
 
+class LibVars(Lib):
+
+    def addLibFuncs(self):
+        self.locls.update({
+            'get': self._libVarsGet,
+            'del': self._libVarsDel,
+            'list': self._libVarsList,
+        })
+
+    async def _libVarsGet(self, valu, strip=False):
+        '''
+        Resolve a variable in a storm query
+        '''
+        if strip:
+            valu = valu.lstrip('$')
+
+        if valu not in self.runt.vars:
+            mesg = f'No var with name: {valu}'
+            raise s_exc.StormRuntimeError(mesg=mesg, valu=valu, strip=strip) from None
+
+        return self.runt.vars.get(valu)
+
+    async def _libVarsDel(self, valu, strip=False):
+        '''
+        Unset a variable in a storm query.
+        '''
+        if strip:
+            valu = valu.lstrip('$')
+
+        try:
+            del(self.runt.vars[valu])
```

Comment:
no need to call del directly - `self.runt.vars.pop(valu, None)` is more consistent across the rest of the codebase

Reply:
How do we delete vars from Path objects as well?